applications/StructuralMechanicsApplication/python_scripts/assign_nodal_elements_to_nodes_process.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+
+def Factory(settings, Model):
+    if not isinstance(settings, KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return AssignNodalElementsToNodesProcess(Model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class AssignNodalElementsToNodesProcess(KratosMultiphysics.Process):
+    """This process creates nodeal elements in all the nodes of a model part
+
+    Only the member variables listed below should be accessed directly.
+
+    Public member variables:
+    Model -- the container of the different model parts.
+    settings -- Kratos parameters containing solver settings.
+    """
+    def __init__(self, Model, settings):
+        """ The default constructor of the class
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        Model -- the container of the different model parts.
+        settings -- Kratos parameters containing solver settings.
+        """
+
+        KratosMultiphysics.Process.__init__(self)
+
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "main_model_part_name"           : "Structure",
+            "sub_model_part_name"            : "",
+            "rayleigh_damping"               : false,
+            "interval"                       : [0.0, 1e30]
```

Comment:
why do you use two sets of parameters?
I am (still) really confused as to how this should be used

Reply:
Because you may not want to generate all kind of sitiffnesses or masses, so thats why